src/pipelinedefinition.cpp

```diff
@@ -665,6 +665,8 @@ Status PipelineDefinition::validateNodes(ModelManager& manager) {
         return StatusCode::PIPELINE_MULTIPLE_EXIT_NODES;
     }
 
+    bool isDemultiplexer = false, isMultipleBatchSize = false;
```

Comment:
Minor:
isDemultiplexer -> isDemultiplexerUsed
There is no demultiplexerPipeline just demultiplexer nodes.

isMultipleBatchSize -> maybe isBsGreaterThan1Used?